datahub-web-react/src/app/entityV2/shared/tabs/Dataset/Validations/contract/DataQualityContractSummary.tsx

```diff
@@ -65,7 +65,7 @@ export const DataQualityContractSummary = ({ contracts, showAction = false }: Pr
             ),
         },
         {
-            title: () => <ColumnHeader style={{ display: 'flex', justifyContent: 'center' }}>STATUS</ColumnHeader>,
+            title: () => <ColumnHeader style={{ display: 'flex', justifyContent: 'center' }}>Status</ColumnHeader>,
```

Comment:
this aligns this table with our standard design pattern of giving table headers UpperCamelCase casing